src/components/materialization/create/CollectionConfig.tsx

```diff
@@ -17,21 +17,20 @@ function CollectionConfig() {
                 header={
                     <FormattedMessage id="materializationCreation.collections.heading" />
                 }
-                body={
-                    <>
-                        <CollectionSelector />
-                        {collections.map((collection: any, index: number) => {
-                            return (
-                                <ExpandableResourceConfig
-                                    collectionName={collection}
-                                    id={imageTag.id}
-                                    key={`CollectionResourceConfig-${index}`}
-                                />
-                            );
-                        })}
-                    </>
-                }
-            />
+            >
```

Comment:
Having a `prop` named body does not really make sense when you can just pass children in.